site/src/components/RichParameterInput/RichParameterInput.tsx

```diff
@@ -70,16 +65,16 @@ export const RichParameterInput: FC<RichParameterInputProps> = ({
   onChange,
   parameter,
   initialValue,
-  ...props
+  ...fieldProps
```

Comment:
Why spread this? Isn't it just `id`?

Reply:
Nops, it passes other props. I probably should fix the types to make it clear... Going to make that.